README.md

```diff
@@ -507,6 +508,60 @@ stringData:
   PASSWORD: ********
 ```
 
+###### Additional Information about Customization
+
+You can customize `Secret`'s `metadata` and `stringData` sections.
+
+- In the `metadata`, you can customize only its labels and annotations.
+- You can customize `stringData` or utilize one of the available formatting options as detailed in the [Formats of Service Binding Secrets](#formats-of-service-binding-secrets) section.
```

Comment:
it is not that clear

I would mention explicitly that if the string data is not customized then one of the "predefined" formats is used.

Reply:
@avilupu but it isn't "used" unless a user specified one of the keys.
Technically, the one that is used automatically is just key-value pairs.